.packit.yaml

```
@@ -5,13 +5,25 @@
 downstream_package_name: container-selinux
 upstream_tag_template: v{version}
 
+# The rpm spec file is `always` synced to dist-git,
+# no need to specify it here.
+files_to_sync:
+  - src: rpm/gating.yaml
+    dest: gating.yaml
+  - src: plans/
+    dest: .
+  - src: .fmf/
+    dest: .
```

These are `rsync` commands so it should be:
```suggestion
  - src: plans/
    dest: plans/
    delete: true
  - src: .fmf/
    dest: .fmf/
  - .packit.yaml
```
Also don't forget `.packit.yaml` as well, unless you want it seperately defined?

One thing that I've been bitten by is needing to delete a plan downstream, so make sure you have `delete: true` where needed.